localisation/00_zuwc_united_localisation.csv

```diff
@@ -1156,7 +1156,7 @@ EVTOPTA_WCDRU_325_druid;How did they manage to open a portal to the Emerald Drea
 EVTOPTA_WCDRU_325_human;Curses!;Mal�diction !;Verflucht!;;;;;;;;;;;x
 EVTOPTB_WCDRU_325;It's my fault!;C'est ma faute�!;Es ist mein Fehler!;;;;;;;;;;;x
 WORGEN_PACK;[Root.GetTitledFirstName]'s Pack;Meute de [Root.GetTitledFirstName];[Root.GetTitledFirstName]s Rudel;;;;;;;;;;;x
-EVTDESC_WCDRU_5005;A pack of wild worgen attacked and ruined [FromFrom.GetName]. Those, who have been bitten, were cursed. Now they became worgen and joined their irrestrainable horde.;Une meute de worgens sauvages a attaqu� et ruin� [FromFrom.GetName]. Ceux qui ont �t� mordus ont �t� maudits. Ils sont maintenant devenus des worgens et ont rejoint leur horde irr�sistible.;Ein Rudel wilder Worgen wurde angegriffen und ruiniert[FromFrom.GetName]. Diejenigen, die gebissen wurden, sind verflucht. Nun wurden sie zu Worgen und schlossen sich ihrer unaufhaltsamen Schar an.;;;;;;;;;;;x
+EVTDESC_WCDRU_5005;A pack of wild worgen attacked and ruined [FromFrom.GetName]. Those, who have been bitten, were cursed. They eventually became worgen and joined their irrestrainable horde.;Une meute de worgens sauvages a attaqu� et ruin� [FromFrom.GetName]. Ceux qui ont �t� mordus ont �t� maudits. Ils sont maintenant devenus des worgens et ont rejoint leur horde irr�sistible.;Ein Rudel wilder Worgen hat angegriffen und verw�stete [FromFrom.GetName]. Diejenigen, die gebissen wurden, sind verflucht. Sie wurden zu Worgen und schlossen sich dem unaufhaltbaren Rudel an.;;;;;;;;;;;x
```

Comment:
Rewrote the second sentence as the commas made things awkward
```suggestion
EVTDESC_WCDRU_5005;A pack of wild worgen attacked and ruined [FromFrom.GetName]. Those who were bitten were inflicted with the curse. They eventually became worgen and joined their irrestrainable horde.;Une meute de worgens sauvages a attaqu� et ruin� [FromFrom.GetName]. Ceux qui ont �t� mordus ont �t� maudits. Ils sont maintenant devenus des worgens et ont rejoint leur horde irr�sistible.;Ein Rudel wilder Worgen hat angegriffen und verw�stete [FromFrom.GetName]. Diejenigen, die gebissen wurden, sind verflucht. Sie wurden zu Worgen und schlossen sich dem unaufhaltbaren Rudel an.;;;;;;;;;;;x
```

Reply:
i suppose these were sort of oxford commas, not really necessary and only style, right? used your version however